docs/detections/prebuilt-rules/rule-details/application-added-to-google-workspace-domain.asciidoc

```diff
@@ -49,11 +49,11 @@ Applications can be added to a Google Workspace domain by system administrators.
 
 ==== Investigation guide
 
-**Config**
+*Config*
 
 The Google Workspace Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule.
 
-**Important Information Regarding Google Workspace Event Lag Times**
+ *Important Information Regarding Google Workspace Event Lag Times*
```

Comment:
You can probably put this content in a **NOTE** admonition block and replace "NOTE" with "IMPORTANT". Instructions are [here](https://github.com/elastic/docs/blob/926d0efb3cf0384c2797c381e38c07a5a8761c0a/README.asciidoc#admonition-blocks).

Reply:
oooh, will consider this or code block for investigation guides, thanks!